### ISSUE TITLE
Sequence Animation cannot be shared between widgets

### ISSUE THREAD

**Issue by 20Tauri** [kivy/kivy#6293]
### Versions

* Python: 3.6.5, 2.7
* OS: W10, Ubuntu
* Kivy: 1.10.1 with have_properties_to_animate fix
* Kivy installation method:

### Description

When a sequence animation is used to animate several object, the `self.anim1.unbind(on_complete=self.on_anim1_complete)` prevent the other widget to continue their animation

### Code and Logs

```python

'''
Widget animation
================

This example demonstrates creating and applying a multi-part animation to
a button widget. You should see a button labelled 'plop' that will move with
an animation when clicked.
'''

import kivy
kivy.require('1.0.7')

from kivy.animation import Animation
from kivy.app import App
from kivy.uix.button import Button
from kivy.uix.floatlayout import FloatLayout


# create an animation object. This object could be stored
# and reused each call or reused across different widgets.
# += is a sequential step, while &= is in parallel
animation = Animation(size=(500, 500))
animation += Animation(size=(100, 50))
animation.repeat = True

class TestApp(App):
    def animate(self, instance):

        # apply the animation on the button, passed in the "instance" argument
        # Notice that default 'click' animation (changing the button
        # color while the mouse is down) is unchanged.
        animation.start(instance)

    def build(self):
        # create a button, and  attach animate() method as a on_press handler
        layout = FloatLayout()
        layout.add_widget( Button(pos=(0, 0), size_hint=(None, None), text='plop', on_press=self.animate) )
        layout.add_widget( Button(pos=(300, 0), size_hint=(None, None), text='plop', on_press=self.animate) )
        return layout


if __name__ == '__main__':
    TestApp().run()

```

**Comment by gottadiveintopython** [comment]
#5926 probably fix the issue iirc.

**Comment by 20Tauri** [comment]
**No it's another issue**,

I'm a reporter of this problem (https://github.com/kivy/kivy/issues/5443)... and it's another problem.
In fact, for the previous problem,  you have created the pull request according my analyse.

Now, the problem is only with Sequence... The binding is incorrect... I have made my own patch (removing the unbind and dding the binding on creation like the others). I don't understand why the code is so complicated... Perhaps, I miss an usecase it's work for me.

Note, that your fix #5926 don't seem merged in master (https://github.com/kivy/kivy/blob/master/kivy/animation.py).

**Comment by gottadiveintopython** [comment]
Did you actually tried #5926? That PR actially fix the issue iirc.

> Note, that your fix #5926 don't seem merged in master 

Yes, so you have to merge it manually. Or you can try [this](https://github.com/kivy-garden/garden.animationbuilder/blob/master/animationbuilder/_replacement_of_the_animation_class.py), which the PR already merged.

**Comment by 20Tauri** [comment]
No I haven't try the fix. But you are right, it should fix this issue.

Kivy will be better if all validated fix are applied and if fix releases will be delivered more ofen.
This will avoid useless report and avoid users to loss time with already solved issues

Thanks for your help, I will try your version

**Comment by 20Tauri** [comment]
Your version is great...
I really not understand why the core team have added it in the 1.11 release.
